packages/bsp/common/usr/lib/armbian/armbian-firstlogin

```diff
@@ -223,7 +223,7 @@ add_user()
 		if [[ $first_input == $second_input ]]; then
 			result="$(cracklib-check <<<"$password")"
 			okay="$(awk -F': ' '{ print $2}' <<<"$result")"
-			if [[ "$okay" == "OK" ]]; then
+			if [[ "$okay" == "OK" || "$okay" == "" ]]; then
```

Comment:
I'd prefer to conditionally check and cracklib,  instead of short-circuit the test and make noise in console

so more like
```bash
if [[ $first_input == $second_input ]]; then
    if [[ $(eval which cracklib-check) -eq 0 ]]; then
	result="$(cracklib-check <<<"$password")"
 			okay="$(awk -F': ' '{ print $2}' <<<"$result")"
 			if [[ "$okay" == "OK" ]]; then
 			
 ```

Reply:
This is better!

Reply:
@lanefu Your solution like an idea is very interesting. But your code doesn't work. 
Please rewrite your solution!

[cr_lanefu.zip](https://github.com/armbian/build/files/6792810/cr_lanefu.zip)

Reply:
was providing you a rough example on how to solve.